src/Authenticator/LoginHandler.php

```diff
@@ -98,9 +98,6 @@ public function doLogin($data, MemberLoginForm $form, HTTPRequest $request)
             return parent::doLogin($data, $form, $request);
         }
 
-        // Enable sudo mode. This would usually be done by the default login handler's afterLogin() hook.
-        $this->getSudoModeService()->activate($request->getSession());
```

Comment:
framework already handles this during login. No idea why this was here in the first place, it's definitely redundant.